go-controller/pkg/node/node.go

```diff
@@ -35,26 +39,36 @@ import (
 	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/util"
 )
 
+const (
+	egressFirewallDNSDefaultDuration        = 30 * time.Minute
+	egressfirewallCRD                string = "egressfirewalls.k8s.ovn.org"
```

Comment:
Can't these be imported from the API in `pkg/crds`?

Reply:
as far as I can tell we can only import the group name